### ISSUE TITLE
Packaging port for Gentoo Linux

### ISSUE THREAD

**Issue by rozniak** [rozniak/xfce-winxp-tc#200]
Issue to track progress on packaging port to Gentoo Linux via `portage` package manager

**Comment by rozniak** [comment]
Having thought about this lately, because I am somewhat more familiar with package managers. As someone who hasn't used Gentoo but has done a smidgen of reading up on it - does this issue make sense?

From what I understand, it doesn't really make sense to produce binary packages for Gentoo. The main purpose I gather is the ability to compile exactly what you like, how you like (with respect to things like optimisations on the user's particular CPU).

This does beg the question for how to provide 'installation' on Gentoo... whilst it is easy to add the ability to specify additional compiler options and whatnot, there is also the POV of having to set everything up manually. I suppose that is part of the appeal, but this may mean a lack of things like the OOBE since that's the 'automated' setup.

Would be interested on Gentoo users' thoughts on what they would like/expect from the project.

**Comment by ernes2306** [comment]
Yeah, here in gentoo we don´t need the binary, we need the ebuild which is basicaly a set of instructions so portage can build, install and maintain the package. Gentoo Wiki explains it here: [https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds](url)

**Comment by rozniak** [comment]
Had a thought about this - probably in some ways the install will need to be akin to the AUR.

Need to whittle down the specifics, but essentially build+package to be done using the system portage. As I had thought with installation via the AUR, it makes sense to separate out post-install to an executable that can be launched manually. Aka the compile/install by the user takes the place of the typical setup process, and the OOBE can be manually called up to complete configuration tasks.

**Comment by rozniak** [comment]
Pillow has been added to the system package dependencies and I have dropped the requirement for pip. This should mean it's possible to sort out EBUILD stuff without requiring network access.
